metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -336,7 +336,7 @@ def _convert_value(param: parameters.Parameter):
             show_status(run_id, kfp_run_url, obj.echo, succeeded)
         elif wait_for_completion:
             response = flow._client.wait_for_run_completion(
-                run_pipeline_result.run_id, timeout=500
+                run_pipeline_result.run_id, timeout=workflow_timeout
```

Comment:
that's the wrong parameter, workflow_timeout is for the timeout of the workflow.

Reply:
not a wait_for_run_completion timeout..

Reply:
I'm totally fine to change that in a follow up PR.

Reply:
This uses the same `workflow_timeout` for the workflow to time out and for the wait...

Reply:
Isn't that fine though? If the workflow times out shouldn't the wait also timeout? Maybe we want to add some padding (ie. add a minute or so the wait) but in theory the `wait_for_completion` should be bounded by the `workflow_timeout` unless I'm missing something.